x-pack/test/fleet_api_integration/apis/package_policy/delete.ts

```diff
@@ -12,143 +12,274 @@ export default function (providerContext: FtrProviderContext) {
+            },
+          });
+        packagePolicy = packagePolicyResponse.item;
+      });
+
+      after(async function () {
+        await supertest
+          .post(`/api/fleet/agent_policies/delete`)
+          .set('kbn-xsrf', 'xxxx')
+          .send({ agentPolicyId: agentPolicy.id });
+
+        await supertest
+          .post(`/api/fleet/package_policies/delete`)
+          .set('kbn-xsrf', 'xxxx')
+          .send({ force: true, packagePolicyIds: [packagePolicy.id] });
+      });
+      after(async () => {
+        await getService('esArchiver').unload('x-pack/test/functional/es_archives/empty_kibana');
+        await getService('esArchiver').unload(
+          'x-pack/test/functional/es_archives/fleet/empty_fleet_server'
+        );
+      });
+
+      it('should fail on hosted agent policies', async function () {
+        // update existing policy to hosted
+        await supertest
+          .put(`/api/fleet/agent_policies/${agentPolicy.id}`)
+          .set('kbn-xsrf', 'xxxx')
+          .send({
+            name: agentPolicy.name,
+            namespace: agentPolicy.namespace,
+            is_managed: true,
+          })
+          .expect(200);
+
+        // try to delete
+        const { body: results } = await supertest
+          .post(`/api/fleet/package_policies/delete`)
+          .set('kbn-xsrf', 'xxxx')
+          .send({ packagePolicyIds: [packagePolicy.id] })
+          .expect(200);
+
+        // delete always succeeds (returns 200) with Array<{success: boolean}>
+        expect(Array.isArray(results));
+        expect(results.length).to.be(1);
+        expect(results[0].success).to.be(false);
+        expect(results[0].body.message).to.contain(
+          'Cannot remove integrations of hosted agent policy'
+        );
+
+        // same, but with force
+        const { body: resultsWithForce } = await supertest
+          .post(`/api/fleet/package_policies/delete`)
+          .set('kbn-xsrf', 'xxxx')
+          .send({ force: true, packagePolicyIds: [packagePolicy.id] })
+          .expect(200);
+
+        // delete always succeeds (returns 200) with Array<{success: boolean}>
+        expect(Array.isArray(resultsWithForce));
+        expect(resultsWithForce.length).to.be(1);
+        expect(resultsWithForce[0].success).to.be(true);
+
+        // revert existing policy to regular
+        await supertest
+          .put(`/api/fleet/agent_policies/${agentPolicy.id}`)
+          .set('kbn-xsrf', 'xxxx')
+          .send({
+            name: agentPolicy.name,
+            namespace: agentPolicy.namespace,
+            is_managed: false,
+            force: true,
+          })
+          .expect(200);
+      });
 
-    it('should work for regular policies', async function () {
-      await supertest
-        .post(`/api/fleet/package_policies/delete`)
-        .set('kbn-xsrf', 'xxxx')
-        .send({ packagePolicyIds: [packagePolicy.id] });
+      it('should work for regular policies', async function () {
```

Comment:
this test seems to be the duplicate of the test at line 133

Reply:
Yes looks like I made a typo here, should be fixed